ci-cd/build-nodejs-project/action.yml

```
@@ -45,6 +57,11 @@ runs:
         echo "::group::build-nodejs-project: npm run lint"
         npm run lint
         echo "::endgroup::"
+        if [ ! "$(get-val 'npm-test-e2e')" == 'true' ]; then
```

Er det en logisk feil her? "Hvis ikke den er true så kjør test"

Uansett med når vi kan lene oss på at denne alltid er definert blir dette enklere:
```suggestion
        if [ '${{ fromJSON(inputs.dsb-build-envs).npm-test-e2e }}' == 'true' ]; then
```